### PR TITLE
AC-491 fix: added styles for experiences mobile view

### PR DESCRIPTION
**Pull request by MattMoran12**
### Checklist for Pull Requests

🚨Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository.

- [ ] Make sure your branch matches the naming conventions. (Something like: `AC-152`). Don't request your master!
- [ ] Your branch should only contain changes relevant to the ticket it is named after.
- [ ] Make sure you are making a pull request against the **dev branch** (left side). Also you should start _your branch_ off _our dev_.
- [ ] Check your code additions will fail neither code linting checks nor e2e test (this will be tested automatically when you open your PR).

### Description
- added styles specifically for when the window size is a mobile view 
- This is just a patch! Trevor is creating a mobile view for experiences that will later be implemented

### CODE REVIEW

File: src/components/experiences/feedback-experiences.js [hunk 1]
```diff
@@ -106,15 +106,20 @@ const Actions = styled(DialogActions)`
 const Box = styled(DialogActions)`
   flex-direction: row; 
   align-self: center;  
+  ${window.innerWidth < 960 ? 'margin: 3px' : ''};
+  .gBTTJU {
```
Comment (crimsonsunset) [inline src/components/experiences/feedback-experiences.js:110]: what is this class? most likely its auto generated and this wont hold thru every run of the app

File: src/components/experiences/feedback-experiences.js [hunk 2]
```diff
@@ -90,10 +90,10 @@ export default function FeedbackExperiences(props) {
 
 const Content = styled(DialogContent)`
   overflow-y: visible;
-  padding: 0 24px;
+  padding: ${window.innerWidth < 960 ? '0px' : '0 24px'};
```
Comment (crimsonsunset) [inline src/components/experiences/feedback-experiences.js:93]: wrap ternaries in ()

File: src/components/experiences/initial-experiences.js [hunk 3]
```diff
@@ -33,11 +33,11 @@ export default function InitialExperiences(props) {
     </Content>
      <Actions>
      <RateButton color={'primary'} variant={isPosSelected(true)} onClick={() => handleFeedback(true)}> 
-       <ThumbUpIcon style={{margin: '5%'}} fontSize={'large'}/> 
+       <ThumbUpIcon style={{margin: '5%'}} fontSize={window.innerWidth < 960 ? 'medium' : 'large'}/>
```
Comment (crimsonsunset) [inline src/components/experiences/initial-experiences.js:36]: same

**Review by crimsonsunset — CHANGES_REQUESTED** [review]

File: src/components/experiences/feedback-experiences.js (hunk 1 above)
Comment (MattMoran12) [reply src/components/experiences/feedback-experiences.js]: Yes I believe it was auto generated. I moved the styles in this class one component up and it has the same effect and will work for every run.

File: src/components/experiences/feedback-experiences.js (hunk 2 above)
Comment (MattMoran12) [reply src/components/experiences/feedback-experiences.js:93]: ternaries are now wrapped in parenthesis

File: src/components/experiences/initial-experiences.js (hunk 3 above)
Comment (MattMoran12) [reply src/components/experiences/initial-experiences.js]: ternaries are now wrapped in parenthesis

**Pull request merged**